### ISSUE TITLE
Session I want to monitor is not being logged

### ISSUE THREAD

**Issue by Thunderous-Leader** [billgraziano/xelogstash#64]
I'm currently trying to pull logs from a session I've created. When I view the target data for the session there are over 100 events in there. When I run the program all I get back is the system health and I have removed that from the config. Please see the config below:
```
[filesink]
retain_hours = 12

[app]

[defaults]
sessions = ["sessionICreated"]
look_back = "168h" # one week
rows = 20000 # Process batches of 20000 at a time.  a good compromise.  0 or omitted processes all the rows

timestamp_field_name = "@timestamp"
payload_field_name = "mssql" # all the XE events are under this field


[[source]]
fqdn = "localhost"
```
I've not used .toml files before. Any help would be much appreciated.

**Comment by billgraziano** [comment]
Does `sessionICreated` have a file target?  Or only a ring buffer?  I can only pull from sessions with file targets.

**Comment by Thunderous-Leader** [comment]
It has a file target. I saw the health check had a ring buffer. But I guess from your reply that doesn't matter for this it only needs a file target?

**Comment by billgraziano** [comment]
This software can only read file targets.  system_health has both a ring buffer and file target -- at least on the server I'm looking at right now.  Are there any errors in the logs?  

What kinds of events are in your session?

The other option is to check the `xestate` folder.  That has a file for each server and session that shows the file name and last offset read.  That should be current with what a direct query of the target file shows.

**Comment by Thunderous-Leader** [comment]
I have checked the session properties. The target type is event_file. Is that correct?

In the xestate directory, the files for SessionICreated.state and SessionICreated.state.0 both show the path to the .xel file I'm trying to read a number e.g. 82606592 then the word 'good'. 

Unfortunately, I can't state the events I'm trying to read in my session.

I tried running sqlwriter -debug -v but it came back as the command not recognized.
